src/js/types/__tests__/cardano.js

```diff
@@ -218,11 +224,74 @@ export const cardanoSignTransaction = async () => {
                     },
                 },
                 amount: '3003112',
+                tokenBundle: [
+                    {
+                        policyId: 'aaff00..',
+                        tokenAmounts: [{ assetNameBytes: 'aaff00..', amount: '3003112' }],
+                    },
+                ],
+            },
+        ],
+        certificates: [
+            {
+                type: 0,
```

Comment:
In some other place we had a nice comment on what the constant means, it would be suitable here, too.

Reply:
These are just the type tests. All the fields just contain some default data for their type so I don't think the explanation is needed here.